addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -213,11 +213,22 @@ const wSnippetMenu = weSnippetEditor.SnippetsMenu.extend({
         if (HighlightOptionEl) {
             HighlightOptionEl.dataset.textSelector = HighlightOptionEl.dataset.selector;
         }
-        
+
         // TODO remove in master: see snippets.xml
         $html.find('we-checkbox[data-dependencies="!footer_copyright_opt"]')[0]?.remove();
         $html.find('[data-name="header_language_selector_none_opt"]')[0]?.remove();
         $html.find('we-select[data-dependencies="!header_language_selector_none_opt"]')[0]?.removeAttribute("data-dependencies");
+
+        // Replace the "src" attribute of iframes with "data-src" to prevent the
+        // iframes from loading in the snippet menu. This is a fix in stable
+        // that is no longer needed in later versions, where the snippets' HTML
+        // code is no longer rendered in the snippet menu.
+        const iframeEls = $html.find("[data-snippet] iframe[src]");
+        for (const iframeEl of iframeEls) {
+            const iframeSrc = iframeEl.getAttribute("src");
+            iframeEl.removeAttribute("src");
+            iframeEl.dataset.src = iframeSrc;
+        }
```

Comment:
To be a bit more stable friendly: we should consider that some people might already have done that, e.g. using an iframe with data-src directly and wanting to only convert it to src at specific moment with a public widget (like the snippet could be *saved* with data-src and changed to src post loading of the page for visitors). It's something very common to do with iframes etc (we actually do it for our scripts). If someone did that, your update will break that: it will be changed to `src` directly during drag and drop.

It is easy to solve. Here, after you change the `src` into `data-src` also add another marker (a class or a data-attribute) and rely on that during that drag and drop. Actually, even better: do not use `data-src` but something very specific from us. Maybe `data-o-src-on-drop`?